src/Illuminate/Support/Number.php

```diff
@@ -342,6 +343,26 @@ public static function trim(int|float $number)
         return json_decode(json_encode($number));
     }
 
+    public static function roman(int $number): string
+    {
+        if ($number < 1 || $number > 3999) {
+            throw new InvalidArgumentException('Number must be between 1 and 3999');
+        }
+
+        $values = [1000, 900, 500, 400, 100, 90, 50, 40, 10, 9, 5, 4, 1];
+        $symbols = ['M', 'CM', 'D', 'CD', 'C', 'XC', 'L', 'XL', 'X', 'IX', 'V', 'IV', 'I'];
```

Comment:
I wonder what the need for "intermediate"/"compound" numbers (900/CM, 400/CD, 90/XC, 40/XL, 9/IX, and 4/IX) is